docs/user/02-11-examples.md

```diff
@@ -0,0 +1,32 @@
+# Examples
```

Comment:
Can you make the title more descriptive?

Reply:
Corrected to "Sap BTP Service Operator Examples".